modules/howtos/pages/n1ql-queries-with-sdk.adoc

```diff
@@ -188,13 +188,12 @@ the assumption is that the `airline` collection exists within a scope `us`.
 
 [source,golang]
 ----
-myCollection := "airline"
-query := fmt.Sprintf("SELECT x.* FROM %s x LIMIT 10;", myCollection)
-        rows, err := us.Query(query, &gocb.QueryOptions{})
-        // check query was successful
-        if err != nil {
-                panic(err)
-        }
+scope := cluster.Bucket("mybucket").Scope("myscope")
+rows, err := scope.Query("SELECT x.* FROM "airline" x LIMIT 10;", &gocb.QueryOptions{})
```

Comment:
does `airline` need to be in backticks, or do those quotes need to be escaped?

Reply:
It doesn't *need* to be in backticks, it sure needs to be not in quotes though - good catch!